packages/teams-js/test/public/stageView.spec.ts

```diff
@@ -48,69 +48,64 @@ describe('stageView', () => {
 
           await utils.initializeWithContext(frameContext);
 
-          expect(() => stageView.open(stageViewParams)).toThrowError(
+          await expect(() => stageView.open(stageViewParams)).rejects.toThrowError(
             `This call is only allowed in following contexts: ["content"]. Current context: "${frameContext}".`,
           );
         });
       });
 
     it('should not allow a null StageViewParams parameter', () => {
-      utils.initializeWithContext('content');
-
-      expect(() => stageView.open(null)).toThrowError('[stageView.open] Stage view params cannot be null');
+      expect.assertions(1);
+      utils.initializeWithContext('content').then(() => {
+        expect(() => stageView.open(null)).rejects.toThrowError('[stageView.open] Stage view params cannot be null');
+      });
     });
 
     describe('v1', () => {
```

Comment:
I think the  v1 unit tests can just be removed actually since there is just version of the API now.